src/views/TeamStatisticsPanel/TeamStatisticsPanel.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Statistic from '../Statistic';
```

Comment:
```js
import Statistic from 'components/Statistic';
```
We should default to `components/...` unless the component is only used by the importing module.